### PR TITLE
feat: [#187790535] enable ticket brokers and resellers license status check

### PR DESCRIPTION
**Pull request by AnJuHyppolite**
## Description

Enable ticket broker license status check.

### Ticket

This pull request resolves [#187790535](https://www.pivotaltracker.com/story/show/#187790535).

### Steps to Test

1. Access the site as a Poppy or Dakota as a Health Club (industry).
2. Once you are directed to the dashboard, click the `Business Profile` link.
3. In the business profile, respond `yes` to the non-essential question, _Do you plan to resell admission tickets to places of entertainment and charge more than the original price?_, and save your response.
4. Click the `Register Your Business as a Ticket Broker (Ticket Reseller)` task in the roadmap.
5. Enter the contact information for a business.
6. The license status will display for the business.

## Code author checklist

- [x] I have rebased this branch from the latest main branch
- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see [CMS Additions in Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po/edit#heading=h.fu5jdsrcqxbh))
- [x] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

File: api/src/client/dynamics/license-status/DynamicsLicenseApplicationIdClient.ts [hunk 1]
```diff
@@ -72,6 +72,7 @@ const licenseTypeToLicenseId: Record<string, string> = {
   "Health Care Services": "3ac8456a-53df-eb11-bacb-001dd8028561",
   "Health Club": "af8e3564-53df-eb11-bacb-001dd8028561",
   Telemarketing: "7e957057-53df-eb11-bacb-001dd8028561",
+  "Ticket Brokers": "538e3564-53df-eb11-bacb-001dd8028561",
```
Comment (jphechter) [inline api/src/client/dynamics/license-status/DynamicsLicenseApplicationIdClient.ts:75]: I don't know if you intended to do this, but I love that this list is alphabetized.

File: content/src/roadmaps/industries/retail.json [hunk 2]
```diff
@@ -49,16 +49,15 @@
     }
   ],
   "name": "Retail",
-  "nonEssentialQuestionsIds": [
-    "ticket-broker-resell"
-  ],
+  "nonEssentialQuestionsIds": ["ticket-broker-resell"],
```
Comment (jphechter) [inline content/src/roadmaps/industries/retail.json:52]: Not your fault, but I thought this ticket ID was a typo at first.

**Review by jphechter — APPROVED** [review]
I think this all looks good. If you're pulling data and have stuff in the Test Licenses sheet, this works for me!

**Review by cbrunefearless — APPROVED** [review]
LGTM

**Pull request merged**